galaxy/main/migrations/0130_import_task_collection_support.py

```diff
@@ -0,0 +1,37 @@
+# Generated by Django 2.1.7 on 2019-03-08 15:46
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('main', '0129_collection_quality_score_updates'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='importtask',
+            name='artifact_id',
+            field=models.CharField(blank=True, max_length=100, null=True),
```

Comment:
Why artifact_id has `CharField` type?

Reply:
I had referenced the celery_task_id being CharField, but can change to `IntegerField`